packages/connect-examples/expo-playground/app/services/previewHardwareParams.ts

```
@@ -0,0 +1,76 @@
+import { getHDPath, getScriptType } from '@onekeyfe/hd-core';
```

ESLint

[error] Resolve error: typescript with invalid interface loaded as resolver

(import/namespace)